ui/src/types/index.ts

```
@@ -137,4 +146,28 @@ export interface PlayerProps {
+        playerId?: string;
+        address?: string;
+        amount?: string;
+        seat?: number;
+        timestamp?: string;
+        round?: string;
+        index?: number;
+        [key: string]: any;
+    }>;
+    isLoading: boolean;
+    error: Error | null;
+    refresh: () => Promise<void | TexasHoldemStateDTO | undefined>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)